### PR TITLE
transmission-daemon: new package at 2.94

### PR DESCRIPTION
**Pull request by ioraff**
Read: <getkiss.org/guidestones.txt>

## Description of package
Torrent daemon and CLI remote

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/transmission-daemon/build [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    --enable-cli \
+    --enable-daemon \
+    --enable-utp \
+    --without-systemd-daemon \
+    --without-gtk
+
+make CXXFLAGS="$CXXFLAGS -std=c++11"
+make -C daemon DESTDIR="$1" install
+
+rm -f "$1/usr/share/transmission/web/LICENSE"
```
Comment (dylanaraps) [inline community/transmission-daemon/build]: Remove this.

File: community/transmission-daemon/build [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --disable-nls \
+    --enable-cli \
+    --enable-daemon \
+    --enable-utp \
+    --without-systemd-daemon \
+    --without-gtk
+
+make CXXFLAGS="$CXXFLAGS -std=c++11"
```
Comment (dylanaraps) [inline community/transmission-daemon/build]: `make -C daemon` perhaps?

Also, why the `-std=c++11`?

File: community/transmission-daemon/build [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sbindir=/usr/bin \
+    --mandir=/usr/share/man \
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --disable-nls \
+    --enable-cli \
```
Comment (dylanaraps) [inline community/transmission-daemon/build]: Does this install with `make -C daemon intall`?

File: community/transmission-daemon/depends [hunk 4]
```diff
@@ -0,0 +1,3 @@
+pkgconf make
```
Comment (dylanaraps) [inline community/transmission-daemon/depends]: Remove this line.

File: community/transmission-daemon/build (hunk 2 above)
Comment (ioraff) [reply community/transmission-daemon/build]: > `make -C daemon` perhaps?
Needs to build libs first, so no.
> Also, why the `-std=c++11`?
Taken from another build, turns out unnecessary.

Comment (ioraff) [reply community/transmission-daemon/build]: > `make -C daemon` perhaps?

need to build libs first, so no

> Also, why the `-std=c++11`?

turns out unnecessary.

File: community/transmission-daemon/build (hunk 3 above)
Comment (ioraff) [reply community/transmission-daemon/build]: removed. `cli` is different from daemon/remote.

**Pull request merged**